packages/server/package.json

```diff
@@ -8,7 +8,7 @@
     "postbuild": "node ./dist/build",
     "watch": "yarn --silent && tsc --build --watch",
     "start": "node dist/index.js",
-    "dev": "yarn --silent && ts-node-dev --inspect --inspect-publish-uid=http --transpile-only src/index.ts"
+    "dev": "yarn --silent && ts-node-dev --debounce 500 --transpile-only src/index.ts"
```

Comment:
We were starting an inspector on the same port every time which prevented two instances of the server to run on yarn dev at the same time. I initially did this so we could attach the debugger all the time, but I ended up never using that so I removed it here. The debouce of 500ms prevents the server from restarting multiple times if you do CTRL + S a lot